### PR TITLE
Use s3 urls for media link maps downloads

### LINKED ISSUE #1774: switch JS link maps to pull directly from S3
Right now we serve the topic link maps via a proxy in `server/topic/maps.py`. This is unnecessary. We should have the JS code source the maps for the user directly from the `url` that the list results now include. This will require changing `DownloadMapFiles` to link directly the `url` for download.

### PR DESCRIPTION
**Pull request by dsjen**
Fixes #1774
* Removes `/api/topics/<topics_id>/map-files/<timespan_maps_id>` endpoint
* Removes `/topics/:topicId/media-map/:mediaMapId` route and associate views on the frontend
* Full text links to the new urls (individual download & view links are removed)

**Previous list**
old

**New list**
new

### CODE REVIEW

File: server/views/topics/apicache.py [hunk 1]
```diff
@@ -478,16 +478,6 @@ def topic_platform_info():
     return mc.topicInfo()
 
 
-def topic_media_map_list(topics_id, timespans_id):
```
Comment (dsjen) [inline server/views/topics/apicache.py:481]: Oops! I was a little overzealous in my deleting. I think this is in fact needed. Hold on...

**Comment by hroberts** [comment]
I don't actually store a meaningful size, so we should remove the (0MB) part.

File: src/components/topic/summary/export/DownloadMapFiles.js [hunk 2]
```diff
@@ -37,28 +33,17 @@ const DownloadMapFiles = (props) => {
...
-              </a>
-            )}
+            <a href={mmf.url} target="_blank" rel="noopener noreferrer">
+              {localMessages[`rename_${mmf.options.color_by}`] ? (<FormattedMessage {...localMessages[`rename_${mmf.options.color_by}`]} />) : mmf.options.color_by }
+              <FormattedMessage
+                {...localMessages.link}
+                values={{
+                  type: mmf.options.color_by,
+                  format: mmf.format,
+                  size: intl.formatNumber(mmf.content_length / 1000000, { maximumFractionDigits: 2 }),
```
Comment (dsjen) [inline src/components/topic/summary/export/DownloadMapFiles.js:43]: TODO: remove size

File: src/components/topic/summary/export/DownloadMapFiles.js [hunk 3]
```diff
@@ -2,25 +2,21 @@ import PropTypes from 'prop-types';
...
   helpIntro: { id: 'topic.summary.mapDownload.help.intro', defaultMessage: '<p>Anayzing the content in this topic as a network can help reveal clusters of content all using the same narrative.  Links Maps show how media sources are linking to each other.</p>' },
   helpTextDetails: { id: 'topic.summary.mapDownload.help.details', defaultMessage: '<p>Link maps are generated automatically by our system. If you don\'t see any here, make a small change and create a new version. That new version will include link maps for each timespan automatically.</p>' },
   none: { id: 'topic.summary.mapDownload.none', defaultMessage: '<p><b>No maps available for this timespan.</b></p>' },
   unsupported: { id: 'topic.summary.mapDownload.unsupported', defaultMessage: 'Sorry, but we can\'t generate link maps when you are using a query filter.  Remove your "{q}" query filter if you want to generate these maps.' },
-  link: { id: 'topic.summary.mapDownload.link', defaultMessage: ' in {format} format ({size}MB) - ' },
+  link: { id: 'topic.summary.mapDownload.link', defaultMessage: ' in {format} format ({size}MB)' },
```
Comment (dsjen) [inline src/components/topic/summary/export/DownloadMapFiles.js:12]: TODO: remove size

**Pull request merged**